Doc/whatsnew/3.15.rst

```diff
@@ -121,6 +121,13 @@ Deprecated
 Removed
 =======
 
+sysconfig
+---------
+
+* The *check_home* parameter of :func:`sysconfig.is_python_build`.
```

Comment:
Please use a complete sentence ("Removed the ...") so when this list is longer, and following a link midway, it's clear what's going on. 

Compare:

https://docs.python.org/3.14/whatsnew/3.14.html#id7

Reply:
Ah! I think I picked up the sentence from the pending deprecation file. My bad, I'll amend it!

Reply:
Actually, it's for most of my entries (there was no entry and I forgot to check the 3.14 page...)